### PR TITLE
feat: text spacy encoder

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,25 +53,25 @@ or in the `.yml` config.
...
+	pip install git+https://github.com/jina-ai/executor-text-spacy-encoder.git
 	```
 
-1. Use `jinahub-MY-DUMMY-EXECUTOR` in your code
+1. Use `jinahub-spacy-text-encoder` in your code
 
 	```python
 	from jina import Flow
-	from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+	from jinahub.encoder.spacy_text_encoder import SpacyTextEncoder
```
Comment (JoanFM) [inline README.md:72]: I think this is not accurate, look at the `setup.py`

Comment (Kelton8Z) [reply README.md:72]: removing the 'text' in the path following 2.0 executor guideline

Comment (JoanFM) [reply README.md:72]: Then change it in setup and in the tests

**Comment by Kelton8Z** [comment]
LGTM

**Review by Kelton8Z — APPROVED** [review]
LGTM

**Pull request merged**